tests/integration/helpers/project.rs

```diff
@@ -41,3 +41,13 @@ impl Project {
         self.path().join(path).exists()
     }
 }
+
+pub fn read(path: &Path) -> String {
```

Comment:
do you know that there is [`fs::read_to_string()`](https://doc.rust-lang.org/std/fs/fn.read_to_string.html) doing exactly this?

Reply:
Yeah, the `read` function is completely useless here